### PR TITLE
update vars in engine.evaluator

### PR DESCRIPTION
**Pull request by chillaq**
# Python SDK

### CODE REVIEW

File: splitio/engine/evaluator.py [hunk 1]
```diff
@@ -43,51 +43,51 @@ def _evaluate_treatment(self, feature, matching_key, bucketing_key, attributes,
         :param attributes: An optional dictionary of attributes
         :type attributes: dict
 
-        :param split: Split object
+        :param feature_flag: Split object
```
Comment (gthea) [inline splitio/engine/evaluator.py:46]: Double checking, is `Split` a type or should it also be changed?

File: splitio/engine/evaluator.py [hunk 2]
```diff
@@ -128,24 +128,24 @@ def evaluate_features(self, features, matching_key, bucketing_key, attributes=No
...
         """
         Evaluate the feature considering the conditions.
 
         If there is a match, it will return the condition and the label.
         Otherwise, it will return (None, None)
 
-        :param split: The split for which to get the treatment
-        :type split: Split
+        :param feature_flag: The feature flag for which to get the treatment
+        :type feature_flag: Split
```
Comment (gthea) [inline splitio/engine/evaluator.py:148]: Double checking, is `Split` a type or should it be changed?

Comment (chillaq) [reply splitio/engine/evaluator.py:148]: It is a class, I put the task to change it in "Could" list

File: splitio/engine/evaluator.py (hunk 1 above)
Comment (chillaq) [reply splitio/engine/evaluator.py:46]: It is a class, I put the task to change it in "Could" list

**Pull request merged**